roles/ostree_admin_unlock/tasks/main.yml

```diff
@@ -7,7 +7,8 @@
 - name: Unlock system
   command: ostree admin unlock
 
-- include: '../../rpm_ostree_status/tasks/main.yml'
+- include_role:
```

Comment:
Let's use import_role/import_task as much as possible and only use include_role/include_task when necessary.